cmd/opm/validate/validate.go

```diff
@@ -0,0 +1,48 @@
+package validate
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/config"
+)
+
+func NewConfigValidateCmd() *cobra.Command {
+	validate := &cobra.Command{
+		Use:   "validate <directory>",
+		Short: "Validate the declarative index config",
+		Long:  "Validate the declarative config JSON file(s) in a given directory",
+		RunE:  configValidate,
+	}
+
+	validate.Flags().BoolP("debug", "d", false, "enable debug log output")
+	return validate
+}
+
+func configValidate(cmd *cobra.Command, args []string) error {
+	debug, err := cmd.Flags().GetBool("debug")
+	if err != nil {
+		return err
+	}
+
+	directory := args[0]
```

Comment:
We should probably have some check above in `NewValidateCmd()` (eg pass `Args:  cobra.ExactArgs(1)` to &cobra.Command{}) to make sure we're not panicking here.

Reply:
We had `Args: cobra.ExactArgs(1)` before - not sure if that was removed in the last force push? But yeah, 100% agree we need to add that, or check the length of the arguments list if we're blindingly indexing the first element like this.